CHANGELOG.md

```diff
@@ -1,3 +1,313 @@
+## [Sidero 0.3.1](https://github.com/talos-systems/sidero/releases/tag/v0.3.1) (2021-08-05)
+
+Welcome to the v0.3.1 release of Sidero!
+
+
+
+Please try out the release binaries and report any issues at
+https://github.com/talos-systems/sidero/issues.
+
+### ServerClass `any` and Environment `default`
+
+Sidero now creates ServerClass `any` which matches all servers.
+
+Environment `default` is created which will supply Talos release that Sidero was built with, as well as default kernel flags.
+
+
+### Application Renaming
+
+The layout of this repository, as well as the naming of the two apps that make up Sidero have changed.
+In an effort to give better clarity to users, there are two apps:
+
+- caps-controller-manager: Responsible for all things related to the Cluster API infrastructur provider
+- sidero-controller-manager: Responsible for all things related to Sidero's bare metal integrations.
+
+It should also be noted that container image names, as well as the kustomize code has been edited to match.
+
+
+### Boot from Disk Methods
+
+If the server is configured to PXE boot by default, it might hit the Sidero iPXE server after Talos install, so Sidero has to force the
+server to boot from disk.
+
+Sidero 0.2 supports default method via iPXE `exit` command, but this command doesn't always work
+([details](https://ipxe.org/appnote/work_around_bios_halting_on_ipxe_exit)).
+
+Sidero 0.3 adds support for two additional methods:
+
+* `http-404` force HTTP 404 response from iPXE server
+* `ipxe-sanboot` uses `sanboot` command to boot from the first disk
+
+Variable `SIDERO_CONTROLLER_MANAGER_BOOT_FROM_DISK_METHOD` controls this setting.
+
+
+### Install and Upgrade Variables
+
+Some aspects of Sidero installation can now be [controlled](https://www.sidero.dev/docs/v0.3/getting-started/installation/) via environment variables or `clusterctl` variables.
+
+
+### IPMI Discovery and Automatic Setup
+
+Sidero agent by default discovers BMC (IPMI) endpoint when it first runs on the server and provisions user for Sidero automatically.
+This feature enables completely automated server and IPMI discovery on the agent boot.
+Variable `SIDERO_CONTROLLER_MANAGER_AUTO_BMC_SETUP` can be used to disable this feature during install or upgrade of Sidero.
+
+Additionally Sidero supports storing IMPI credentials in Kubernetes secrets referencing them from the Server object.
+
+**
+Note that this functionality is known to have issues with iDRAC currently. This can be worked around by updating the `<server-uuid>-bmc` secret that is created in the `sidero-system` namespace with credentials of an existing iDRAC admin user or by disabling the feature altogether with the aforementioned environment variable.
```

Comment:
@rsmitty probably we should remove this and regen the release?

Reply:
oh snap, good catch. thx.

Reply:
kk, fixed.